apps/cuda_mat_mul/Makefile

```diff
@@ -3,6 +3,9 @@ include ../support/Makefile.inc
 BIN ?= bin
 MATRIX_SIZE ?= 1024
 
+CXXFLAGS += -I /usr/local/cuda-8.0/include
```

Comment:
When we do things like this, we should introduce a unique make variable, e.g. CUDA_INCLUDE or some such, so one can write "make CUDA_INCLUDE=~/deps/cuda-8.0/include" to override the path.